axelrod/strategy_transformers.py

```diff
@@ -228,11 +297,21 @@ def dual_wrapper(player, opponent, proposed_action):
     -------
     action: an axelrod.Action, C or D
     """
+
     if not player.history:
-        player.original_player = player.original_class(**player.init_kwargs)
+        player.use_history = []
```

Comment:
@theref could you take a look at this and confirm I've got things right? It's not as simple as just using the history, we need to keep track of the whole state, I don't believe the change written here ensures that.

No matter what we should probably write a test that would catch this.